products/visionos.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Vision OS
+category: os
+tags: apple
+iconSlug: apple
+permalink: /visionos
+releasePolicyLink: https://en.wikipedia.org/wiki/VisionOS#Version_history
+eoasColumn: true
+releaseDateColumn: true
+
```

Comment:
```suggestion

auto:
  methods:
  -   apple: visionos
      regex:
      -   'visionOS\s+(?P<version>\d+)'
      -   'visionOS\s+(?P<version>\d+(?:\.\d+)+)'

```

Did not tested it, but looking at https://support.apple.com/en-us/HT201222 it should work.

Reply:
Tested it, LGTM:

```suggestion
- Item root['releases'] ({}) added to dictionary.
- Item root['versions'] ({'1.2': {'name': '1.2', 'date': '2024-06-10'}, '1.1.2': {'name': '1.1.2', 'date': '2024-04-09'}, '1.1.1': {'name': '1.1.1', 'date': '2024-03-21'}, '1.1': {'name': '1.1', 'date': '2024-03-07'}, '1.0.3': {'name': '1.0.3', 'date': '2024-02-12'}, '1.0.2': {'name': '1.0.2', 'date': '2024-01-31'}, '1': {'name': '1', 'date': '2024-01-31'}}) added to dictionary.
```